src/main/java/uk/co/harieo/ConvenienceLib/menus/MenuFactory.java

```diff
@@ -173,4 +195,16 @@ public void registerDefaultInteractionListener(JavaPlugin plugin) {
 		Bukkit.getPluginManager().registerEvents(interactionListener, plugin);
 	}
 
+	void cleanup(Player player) {
+
+		// remove items for the player
+		Map<Integer, MenuItem> itemsForPlayer = items.row(player.getUniqueId());
```

Comment:
Is it possible that there is no row for this player?

Reply:
fixed 👍